galsim/wcs.py

```diff
@@ -2371,7 +2525,16 @@ def radec_func(self): return self._radec_func
     def origin(self): return self._origin
 
     def _radec(self, x, y, color=None):
-        return self._radec_func(x,y)
+        try:
+            return self._radec_func(x,y)
+        except Exception as e:
+            try:
+                world = [ self._radec(x1,y1) for (x1,y1) in zip(x,y) ]
+            except Exception:  # pragma: no cover
+                raise e  # Raise the original one if this fails, since it's probably more relevant.
```

Comment:
This try except block seems like a better fit for the ones I flagged above (and a few below that).